imod_coupler/drivers/dfm_metamod/dfm_utilities.py

```diff
@@ -0,0 +1,27 @@
+import numpy as np
+from bmi.wrapper import BMIWrapper
+from numpy.typing import NDArray
+
+
+class DfmUtilities:
+    @classmethod
```

Comment:
As discussed before, instead of adding a couple of `classmethod`s that all take the dflow `BMIWrapper`, subclass `BMIWrapper` instead. `mypy` will complain since `BMIWrapper` is untyped, but you can just ignore this as shown here: https://github.com/Deltares/imod_coupler/blob/1dcb67bb252911cb107f95655f75ac84c1320188/imod_coupler/drivers/dfm_metamod/dfm_wrapper.py#L6

Reply:
done